app/scripts/views/mixins/modal-settings-panel-mixin.js

```diff
@@ -45,6 +46,12 @@ define(function (require, exports, module) {
       this.closePanel();
     },
 
+    _closePanelReturnToClients: function () {
+      this.navigate('settings/clients');
```

Comment:
I understand why you did this, and makes sense, but I fear we are going to end up with a generic mixin ending up with a lot of view specific routes and logic. I can easily imagine us adding a `#back` button to a non-avatar related view in the future and not understanding why clicking the button keeps going to `settings/avatar/change`. Can we generalize somehow, either by embedding the next screen view as an attribute on the button, or making the back buttons links whose `href` is set to the next step?